pkg/vdr/panacea_vdr.go

```diff
@@ -44,6 +45,23 @@ func (r *PanaceaVDR) Resolve(didID string, _ ...vdr.DIDMethodOption) (*did.DocRe
 		return nil, fmt.Errorf("failed to parse DID document: %w", err)
 	}
 
+	var vms []did.VerificationMethod
+	for _, vm := range doc.VerificationMethod {
+		var verificationMethod did.VerificationMethod
+		verificationMethod = vm
+		if btcec.IsCompressedPubKey(vm.Value) {
+			pubKey, err := btcec.ParsePubKey(vm.Value, btcec.S256())
+			if err != nil {
+				return nil, fmt.Errorf("invalid secp256k1 public key: %w", err)
+			}
+
+			verificationMethod.Value = pubKey.SerializeUncompressed()
+		}
+		vms = append(vms, verificationMethod)
```

Comment:
```suggestion
	for _, vm := range doc.VerificationMethod {
		if btcec.IsCompressedPubKey(vm.Value) {
			pubKey, err := btcec.ParsePubKey(vm.Value, btcec.S256())
			if err != nil {
				return nil, fmt.Errorf("invalid secp256k1 public key: %w", err)
			}

			vm.Value = pubKey.SerializeUncompressed()
		}
		vms = append(vms, vm)
```
At the beginning of the `for` loop, the `vm` is already copied from an element in the `doc.VerificationMethod` because its type is not pointer.

Reply:
Oh, thank you pointing it out